src/function/pragma/pragma_functions.cpp

```diff
@@ -115,6 +115,14 @@ static void PragmaDisableCheckpointOnShutdown(ClientContext &context, const Func
 	DBConfig::GetConfig(context).options.checkpoint_on_shutdown = false;
 }
 
+static void PragmaEnableWalEncryption(ClientContext &context, const FunctionParameters &parameters) {
+	DBConfig::GetConfig(context).options.enable_wal_encryption = true;
```

Comment:
We try to avoid pragmas nowadays - can you replace this with a setting (e.g. `SET wal_encryption=...`)